src/components/FormPublish.js

```diff
@@ -81,9 +94,8 @@ const FormPublish = ({ onSubmit }) => {
         return errors;
       }}
       onSubmit={(values, { setSubmitting }) => {
-        setTimeout(() => {
-          alert(JSON.stringify(values, null, 2));
-          onSubmit(values);
+        setTimeout(async () => {
```

Comment:
shoudn't we have `setSubmitting(true)` as first action ?

Reply:
also why 400 ?

Reply:
Obviously the `setSubmitting(true)` is done under the curtains. I've tested submitting the form 2 times in a row and I can see that the `isSubmitting` flag works properly.
Still, it's probably a good idea to set it back explicitly each time.

Reply:
About the `400`, no idea why it is there... gonna change it.

Reply:
I've open a specific issue for this one: [issue 12](https://github.com/SocialGouv/carnets/issues/12)